include/sdf/Element.hh

```diff
@@ -810,27 +1011,42 @@ namespace sdf
   template<typename T>
   std::pair<T, bool> Element::Get(const std::string &_key,
                                   const T &_defaultValue) const
+  {
+    sdf::Errors errors;
+    std::pair<T, bool> result = this->Get<T>(errors, _key, _defaultValue);
+    for(auto& error : errors)
+    {
+        sdferr << error.Message();
+    }
```

Comment:
Use `throwOrPrintErrors`

Reply:
As per IM I moved `throwOrPrintErrors` into Error.cc. I opened a new PR with this changes [here](https://github.com/gazebosim/sdformat/pull/1220) since it seems like this changes and further discussion can have it's own scope.

Please review there so it can be used here once merged.

Reply:
Since https://github.com/gazebosim/sdformat/pull/1220 has been merged, `ThrowOrPrintErrors` usage was added in https://github.com/gazebosim/sdformat/pull/1141/commits/8d9f1516267e36903b73e9aed7bfee371b45b1a6.